src/components/Modal/index.js

```diff
@@ -0,0 +1,2 @@
+export { default as CustomModalDialog } from './CustomModalDialog';
+export { default as Modal } from './Modal';
```

Comment:
I am not sure I get it.
* What is the difference between Modal and CustomModalDialog?
* What so **custom** in this modal so it called CustomModalDialog?
* Why do consumers need to consume both of them? Can't we have one configurable Modal?

Reply:
This one is not obvious... 

The problem w/ the About Modal pattern I was having was being able to add an additional class to the `modal-content` div (from Bootstrap). Currently, there is no simple way in the React Bootstrap API it seems other than providing a `dialogComponentClass` and extending the Dialog. I looked at several issues on this and this seems like the most flexible option...

https://github.com/react-bootstrap/react-bootstrap/issues/1940

I prefer to have a JS API rather than having to carry around additional CSS like that issue suggests.

I also tried a lot of other options, but there is not a clear way with current RB apis. My guess is extending the Dialog like that and providing an additional `contentClassName` prop to extend the `modal-content` div with our own classes will be useful. I am exposing CustomModalDialog b/c the About Modal pattern will be common (and maybe there are other scenarios you want to extend `modal-content` too). 

Long story short, I think this is likely a good enhancement upstream if the React Bootstrap team will accept one. This seems the best option for now...

Reply:
Ok i understand the issue now, thanks!

What about implementing our own Modal component, that will receive some custom props and will render the Modal (with the CustomModal) appropriately?

Maybe even a prop call `patternflyModalStyle` that will choose between of them (and can support more styles).

I just prefer to consume only one Modal, flexible as possible.

Reply:
+1 - should be easy enough. I don't think there is any issue w/ always using Custom Dialog but will test it... Custom Dialog will always be more flexible. We can move this to InnerComponents now.

Reply:
:+1:

Reply:
I've made the updates to extend RB Modal w/ CustomDialog by default. This will allow us to always customize the `modal-content` div or anything else in the Modal Dialog we want.